assets/data/host.js

```diff
@@ -0,0 +1,76 @@
+export const fullActivityDescription = `
```

Comment:
This file is really not in a place where I want it.  It is not an asset.

As a `.js` file that we write and maintain, I would like it to be located somewhere in the `js` directory.  Since it's host-specific, it should be in `js/host` somewhere.  There *is* one `.js` file that lives in `assets`, but that is because it is a third-party library that I can't get properly through NPM, and is not code that I maintain at all.

Reply:
I remember you explained it do me & I deleted this file, as stated 100000000 times above haha.